scripts/mc_rtp_launch_record.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2021 The HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+import numpy as np
+from astropy.time import Time
+
+from pyuvdata import UVData
+
+from hera_mc import mc
+from hera_mc.rtp import RTPLaunchRecord
+
+ap = mc.get_mc_argument_parser()
+ap.description = """Add or update the RTP Launch Record for an observation."""
```

Comment:
When will this script be used? I'm not sure I understand the use case.

Reply:
I added some documentation in d2cf9c3. Let me know if it can be improved.